simple-template/main.js

```diff
@@ -3,10 +3,10 @@ customElements.define('my-paragraph',
     constructor() {
       super();
 
-      const template = document.getElementById('my-paragraph');
+      const template = document.getElementById('custom-my-paragraph');
```

Comment:
Same as comment above:

```suggestion
      const template = document.getElementById('custom-p-element');
```